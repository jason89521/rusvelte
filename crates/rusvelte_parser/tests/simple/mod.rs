use crate::test_success;

test_success!();
