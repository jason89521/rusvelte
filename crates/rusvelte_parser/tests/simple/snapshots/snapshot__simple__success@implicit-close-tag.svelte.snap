---
source: crates/rusvelte_parser/tests/simple/mod.rs
expression: root
input_file: crates/rusvelte_parser/tests/simple/inputs/implicit-close-tag.svelte
snapshot_kind: text
---
{
  "type": "Root",
  "css": null,
  "start": 0,
  "end": 16,
  "fragment": {
    "type": "Fragment",
    "nodes": [
      {
        "type": "RegularElement",
        "start": 0,
        "end": 16,
        "name": "p",
        "attributes": [],
        "fragment": {
          "type": "Fragment",
          "nodes": [
            {
              "type": "Text",
              "start": 3,
              "end": 5,
              "raw": "\n\t",
              "data": "\n\t"
            },
            {
              "type": "RegularElement",
              "start": 5,
              "end": 12,
              "name": "span",
              "attributes": [],
              "fragment": {
                "type": "Fragment",
                "nodes": [
                  {
                    "type": "Text",
                    "start": 11,
                    "end": 12,
                    "raw": "\n",
                    "data": "\n"
                  }
                ]
              }
            }
          ]
        }
      }
    ]
  },
  "module": null,
  "instance": null,
  "options": null
}
